### ISSUE TITLE
Lintr error in Rmd and Qmd files

### ISSUE THREAD

**Issue by mihem** [REditorSupport/vscode-R#1419]
**Describe the bug**
When opening a Rmd template, lintr does not work and shows the following error:
```
failed to  run diagnostics: ! in callr subprocess. caused by error: ! linter 'object_length_linter' failed in : path[1]="": no such file or directory
```

**To Reproduce**

1. Open Rmarkdown
2. reprex minimal (or any other template)

**Can you fix this issue by yourself? (We appreciate the help)**

No

// R path for Linux
"r.rterm.linux": "~/miniconda3/bin/radian",

**Screenshots**
With error lens enabled:

![image](https://github.com/REditorSupport/vscode-R/assets/46907494/e79812c5-109a-4f99-a12c-cfdc11ac9c0a)

- OS: vUbuntu 22.04.3 LTS
- VSCode Version: 1.82.1
- R Version: 4.3.1
- vscode-R version: [e.g. 1.2.2]

**Additional context**
I installed the latest dev version of languageserver via
```
install.packages("languageserver", repos = c(
    reditorsupport = "https://reditorsupport.r-universe.dev",
    getOption("repos")
))
```

which did not help.

Also lintr itself works fine, in .R files, but also in when manually calling it

![image](https://github.com/REditorSupport/vscode-R/assets/46907494/a6be5fbb-c58b-4ffb-a8cb-925be6f87b0a)

I found a related topic https://github.com/REditorSupport/vscode-R/issues/1198. However, this topic is supposed to be fixed with a new language server.

Thanks!

**Comment by mcanouil** [comment]
Indeed, this can be seen in Codespaces with https://github.com/mcanouil/quarto-lintr
Although I don't know yet, if it's vscode-R or https://github.com/quarto-dev/quarto.

**Comment by mihem** [comment]
Thanks!

You are right.
I saw that JJ Allaire said that lint was not working in .qmd in Vscode here https://github.com/quarto-dev/quarto/issues/208.
The thread is was talking about earlier https://github.com/REditorSupport/vscode-R/issues/1198 was only referring to Rmarkdown.

So probably this quarto not vscode-R.
.. But would be an important feature.

**Comment by guilhermegarcia** [comment]
Any news on this? Has this been addressed?
